bin-src/ui/messages/info/tracedAffectedFiles.ts

```diff
@@ -38,7 +38,33 @@ export default (
 
   const changed = pluralize('changed files', changedFiles.length, true);
   const affected = pluralize('affected story files', Object.keys(affectedModules).length, true);
-  const summary = chalk`${info} Traced {bold ${changed}} to {bold ${affected}}`;
+  const bailReason = ctx.turboSnap.bailReason
```

Comment:
Can this be run in cases where `ctx.turboSnap` will be undefined, or can we guarantee that will be set in here?

Reply:
@tevanoff 
The function call is gated behind traceChanged
https://github.com/chromaui/chromatic-cli/blob/main/bin-src/lib/getDependentStoryFiles.ts#L260

Also, we call `ctx.turbosnap` all over without the `?` conditional operand.

I'd be happy to add it everywhere if you think it's worth it.

Reply:
Oh ok, all good, just wanted to check. Thanks!